src/AppInstallerCommonCore/ExperimentalFeature.cpp

```diff
@@ -52,6 +52,8 @@ namespace AppInstaller::Settings
                 return userSettings.Get<Setting::EFConfigureSelfElevation>();
             case ExperimentalFeature::Feature::StoreDownload:
                 return userSettings.Get<Setting::EFStoreDownload>();
+            case ExperimentalFeature::Feature::IndexV2:
```

Comment:
_triggers OCD_

Reply:
I'm going to assume that my lazy "Accept Both Changes" click in Code that put my case first is the issue here.  🦪